pricing-app/src/app/admin/orders/orderItem.tsx

```diff
@@ -0,0 +1,49 @@
+import SelectionBox from "@/components/selectionBox";
+import { orderStatus } from "@/mock/data";
+import { Item, Options } from "@/types/type";
+import { FC } from "react";
+
+type Props = {
+  id: string;
+  createdAt: string;
+  updatedAt: string;
+  cost: number;
+  status: string;
+  onHandleUpdateOrder: (id: string) => void;
+  onChangeStatusOrder: (value: Options) => void;
+};
+
+const OrderItem: FC<Props> = ({
```

Comment:
The naming here is a bit confusing, may be OrderItemView: FC<{orderItem: OrderItem, onOrderUpdated:....>